components/stepper/personal-details.js

```diff
@@ -75,198 +80,188 @@ export default function PersonalDetails ({ handleNextStep }) {
     <div>
       <form>
         <div style={{ paddingTop: 50 }}>
-          <div className='row form-group' style={{ padding: '.375rem .75rem' }}>
+          <div className="row form-group" style={{ padding: '.375rem .75rem' }}>
             <h4>
               <span style={{ color: '#13a2dc' }}>Personal</span> Details
             </h4>
           </div>
 
-          <div className='form-group'>
-            <div className='mb-3'>
-              <label className='pdetails'>Recipient's name</label>
+          <div className="form-group">
```

Comment:
I am not really big fan to use bootstrap for this. We don't need bootstrap it's over-engineering for this. Just `flex`was fine. But ok.